Resources/Prototypes/_CM14/Entities/Structures/Walls/walls.yml

```diff
@@ -0,0 +1,88 @@
+# Standart
+- type: entity
+  parent: BaseWall
```

Comment:
Same here, though it would be better to first create a CMBaseWall that all of these walls inherit instead.